Clarify provider URL when running airnode-examples locally (#1561)
api3dao/airnode: packages/airnode-examples/README.md

yarn create-airnode-secrets
```

> > If you are using Docker Desktop (Windows, WSL, Mac, or Linux) and you want to connect to your local hardhat network,

Reviewer: ```suggestion
> If you are using Docker Desktop (on any platform) and you want to connect to your local hardhat network,
```
